src/sif/dynamiccost.cc

```diff
@@ -260,6 +260,13 @@ void DynamicCost::RelaxHierarchyLimits(const float factor, const float expansion
   }
 }
 
+void DynamicCost::RelaxHierarchyLimits(const bool using_bidirectional) {
+    // since bidirectional A* does about half the expansion we can do half the relaxation here
+    const float relax_factor = using_bidirectional ? 8.f : 16.f;
+    const float expansion_within_factor = using_bidirectional ? 4.0f : 2.0f;
```

Comment:
this one seems backwards. shouldnt it be:

`const float expansion_within_factor = using_bidirectional ? 2.0f : 4.0f;`

Reply:
Thanks for catching this!
I must have dumbly copied it from the other place 🤦 
https://github.com/valhalla/valhalla/blob/a2fa483893a6fd9fa129a6353ae21ab5de9f3112/src/valhalla_run_route.cc#L157

It's been fixed now